.changeset/nasty-cloths-make.md

```diff
@@ -0,0 +1,14 @@
+---
+"fluid-framework": minor
+"@fluidframework/map": minor
+---
+
+Moved SharedMap to '@fluidframework/map/legacy'
```

Comment:
Note: The alpha export is about to be renamed '/legacy'.  To simplify, I'm using the soon-to-be-path here instead of the current path.